montage/rdb.py

```diff
@@ -276,6 +276,7 @@ class Round(Base):
     status = Column(String(255), index=True)
     vote_method = Column(String(255))
     quorum = Column(Integer)
+    statistics = Column(Boolean)
```

Comment:
Is there a case where we'd want to prevent statistics from being sent down? If so, it may be easier to simply put a `show_stats` flag in the `flags` column. Then we can kick the can down the road on a migration.

Reply:
Fixed: store `show_stats` in the flags `column`